### DISCUSSION TITLE
How can I detect WSLg availability?

### DISCUSSION THREAD

**Discussion by jmigual** [microsoft/wslg#630] [Q&A] [answered]
I have a script that setups the VcXsrv environment variables on shell startup and I want to disable this behavior if WSLg is available but I'm not sure how can I detect whether WSLg is available or not.

**Comment by Masamune3210** [comment]
You could ask the system if WSL is enabled in the Windows Features and if Windows is above a certain version, or detect if any of the WSLg updates are installed

**Comment by Biswa96** [comment, marked as answer]
Maybe check if `/mnt/wslg` is mounted or not 🤷
